src/sedtrails/transport_converter/physics_lib.py

```diff
@@ -56,22 +56,22 @@ def compute_shear_velocity(bed_shear_stress: np.ndarray, water_density: float) -
     Parameters:
     -----------
     bed_shear_stress : np.ndarray
-        Bed shear stress [N/m²]
+        τ = Bed shear stress [N/m²]
     water_density : float
-        Water density [kg/m³]
+        ρ_w = Water density [kg/m³]
 
     Returns:
     --------
     np.ndarray
-        Shear velocity [m/s]
+        u* = Shear velocity [m/s]
 
     Notes:
     ------
     u* = sqrt(τ / ρ_w)
 
     Reference:
```

Comment:
For automatically formatting references, I suggest use a reference section and add the references as in the following example (this is compatible with the Numpy coding style, and the automatic tools for documentation that we are using)

```rst
    References:
    -----------
    .. [1] Lotka, A. J. (1925). Elements of Physical Biology.
           https://doi.org/10.5962/bhl.title.29508
    .. [2] Volterra, V. (1926). Fluctuations in the abundance of a species.
```